engine/compatmanager/update.go

```diff
@@ -39,7 +40,7 @@ func (db *DB) updateAuthSubjects(ctx context.Context) error {
 	if err != nil {
 		return fmt.Errorf("begin tx: %w", err)
 	}
-	defer tx.Rollback()
+	defer func(tx *sql.Tx) { _ = tx.Rollback() }(tx)
```

Comment:
Use `sqlutil.Rollback` to be consistent with the rest of the app

Reply:
done